src/js/media-chrome-range.ts

```diff
@@ -172,7 +172,8 @@ template.innerHTML = /*html*/ `
       }
     }
 
-    #thumb {
+    #thumb,
+    ::slotted([slot=thumb]) {
```

Comment:
a question here is, do we want to apply default styles to the slotted thumb element? I went ahead with yes but I could see reasons for not wanting that. or part of the styles, like background color, border-radius...

Reply:
we decided to not apply default stylistic styles like color, border, border-radius to the slotted thumb